fjord/heartbeat/analyzer_views.py

```diff
@@ -19,38 +19,65 @@ def hb_data(request, answerid=None):
     """View for hb data that shows one or all of the answers"""
     VALID_SORTBY_FIELDS = ('id', 'received_ts', 'updated_ts')
 
+    ALL_FIELDS = [
+        'id', 'received_ts', 'updated_ts', 'experiment_version',
+        'response_version', 'person_id', 'survey_id',
+        'flow_id', 'question_id', 'question_text', 'variation_id',
+        'score', 'max_score', 'flow_began_ts', 'flow_offered_ts',
+        'flow_voted_ts', 'flow_engaged_ts', 'platform', 'channel',
+        'version', 'locale', 'country', 'build_id', 'partner_id',
+        'profile_age', 'profile_usage', 'addons', 'extra'
+    ]
+
+    DEFAULT_FIELDS = [
+        'id', 'received_ts', 'updated_ts', 'experiment_version',
+        'response_version', 'person_id', 'survey_id', 'flow_id',
+        'question_id', 'variation_id', 'score', 'max_score',
+        'flow_began_ts', 'flow_voted_ts', 'platform', 'channel',
+        'locale'
+    ]
+
     sortby = 'id'
     answer = None
     answers = []
     survey = None
-    showdata = None
+    showdata_filter = None
 
     if answerid is not None:
         answer = Answer.objects.get(id=answerid)
 
     else:
+        # Get fields and split on comma, then get rid of empty fields.
+        fields = request.GET.get('fields', '').split(',')
+
+        # Nix any fields that don't exist
+        fields = [field for field in fields if field and field in ALL_FIELDS]
```

Comment:
Mmm... I can drop the "if field" part because if it's falsey, it won't be in `ALL_FIELDS`. I'll do that.